app/models/input_template_renderer.rb

```diff
@@ -16,84 +16,64 @@ def initialize(template, host = nil, invocation = nil, input_values = nil, previ
     raise Foreman::Exception, N_('Recursive rendering of templates detected') if templates_stack.include?(template)
 
     @host = host
-    @invocation = invocation
     @template = template
+    @invocation = invocation
     @input_values = input_values
     @preview = preview
     @templates_stack = templates_stack + [template]
   end
 
   def render
     @template.validate_unique_inputs!
-    render_safe(@template.template, ::Foreman::Renderer::ALLOWED_HELPERS + [ :input, :render_template, :preview?, :render_error ], :host => @host)
+    Foreman::Renderer.render_template(
```

Comment:
With the latest version of https://github.com/theforeman/foreman/pull/5683/files, I'm getting:

```
Failed to initialize: RuntimeError - Failed rendering template: error during rendering: undefined method `render_template' for Foreman::Renderer:Module
   2:
Did you mean? render_template_to_tempfile
```